recipes/gclib/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: gclib
+  version: "907a9d"
+
+build:
+    number: 0
+    skip: False
+
+source:
+  git_url: https://github.com/gpertea/gclib.git
```

Comment:
This is the syntax they had in https://conda.io/docs/building/meta-yaml.html#source-from-git